docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -201,6 +205,13 @@ The response time for each page will be stored in the same RRD file specified fo
 
 WARNING: You need to delete existing RRD files and let them be recreated with the new list of data sources when you add a ds-name attribute to a page in a sequence that is already storing response-time data.
 
+== Preselect authentication
+When authentication is required, the first HTTP request receives a "401 Unauthorized" response with possible authentication methods. The second request is sent using the selected authentication method.
+
+Using the `preselect-auth` attribute allows you to use a specific authentication right on the first request.
```

Comment:
```suggestion
Using the `preselect-auth` attribute lets you to use a specific authentication on the first request.
```
I prefer "allows" but our style guide requires using "lets" instead of "allows" in this context.